Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/EditorVertexSelection.h

```diff
@@ -291,16 +300,20 @@ namespace AzToolsFramework
     //! EditorVertexSelectionFixed provides selection and editing for a fixed length number of
     //! vertices. New vertices cannot be inserted/added or removed.
     template<typename Vertex>
-    class EditorVertexSelectionFixed : public EditorVertexSelectionBase<Vertex>
+    class EditorVertexSelectionFixed
+        : public EditorVertexSelectionBase<Vertex>
+        , private AzToolsFramework::EditorVertexSelectionRequestBus::Handler
     {
     public:
         AZ_CLASS_ALLOCATOR_DECL
 
-        EditorVertexSelectionFixed() = default;
-        EditorVertexSelectionFixed(EditorVertexSelectionFixed&&) = default;
-        EditorVertexSelectionFixed& operator=(EditorVertexSelectionFixed&&) = default;
```

Comment:
question: Why were these special members removed? I believe it's safe to add them back (by providing a user declared destructor these move operations are implicitly disabled which is why you need to explicitly add them as was done before).